tensorflow/lite/delegates/xnnpack/README.md

```diff
@@ -342,7 +342,7 @@ must be enabled by adding extra Bazel flags when building TensorFlow Lite.
 operators using signed quantization schema. This schema is used by models
 produced by [Model Optimization Toolkit](https://www.tensorflow.org/model_optimization)
 through either post-training integer quantization or quantization-aware
-training. Post-training dynamic range quantization is not supported in XNNPACK.
```

Comment:
This is not dynamic range quantization. Models with dynamic range quantization quantized floating-point inputs to Conv2D/FC/etc, do quantized computations, then dequantize the outputs.

Reply:
I am not quite sure about that. From my experience I have concluded that dynamic-range quantization implies that weights are quantized for storage, but at runtime almost all of them (not all because hybrid kernels are involved) are being dequantized and then all the computations are performed in floating-point. I got this understanding by inspecting converted TFLite models with Netron viewer. Some of my observations are described in [this](https://github.com/tensorflow/tensorflow/pull/48751#issue-867023983) message. Please correct me if I my conclusions are wrong.

Reply:
TFLite converter creates operators with `FP32` inputs/outputs and `INT8` weights when using dynamic-range quantization. These operators are processed by so-called hybrid kernels, which dynamically quantize input and dequantize output. XNNPACK currently doesn't support hybrid schemas, so the models converted with dynamic range quantization will not on the XNNPACK delegate, and it would be confusing to state otherwise.